spec/index.bs

```diff
@@ -16610,7 +16621,7 @@ The [=texel block memory cost=] of each of these formats is the same as its
         <td rowspan=2>8
         <td rowspan=14>{{GPUTextureSampleType/"float"}},<br/>{{GPUTextureSampleType/"unfilterable-float"}}
         <td rowspan=14>4 &times; 4
-        <td rowspan=14>&check;
+        <td rowspan=14>If {{GPUFeatureName/"texture-compression-bc-sliced-3d"}} is enabled
```

Comment:
For these formats it now says:

- `"3d"` => If `"texture-compression-bc-sliced-3d"` is enabled
- Feature => `texture-compression-bc`

That doesn't really make sense, it implies that you have to enable both features to use 3d. Better would be:

- `"2d"` => If `"texture-compression-bc"` is enabled
- `"3d"` => If `"texture-compression-bc-sliced-3d"` is enabled

(Maybe in the future we could make it so that enabling `"texture-compression-bc-sliced-3d"` automatically enables `"texture-compression-bc"`, but that isn't important right now.)

Reply:
@kainino0x should I rename the feature column to 2d? I thought of doing like this because then maybe we also need to list 1d. Would it make sense to just rename as 2d or complete dimensions with 1d, 2d, 3d? Thanks a lot!

Reply:
@kainino0x reading again, wasn't the consensus to require enabling base "-bc" for sliced 3D? If that is not the case, would sliced 3D also declare the enum values? In current PR, the 3D does not add any API intentionally.

Reply:
(not meaning automatic enable but require API user to also enable case -bc)

Reply:
> reading again, wasn't the consensus to require enabling base "-bc" for sliced 3D?

I don't recall discussing it specifically. I was imagining that the two would be totally orthogonal features that each enable these formats for different cases. (This wouldn't be necessary anymore if we made enabling `-bc-sliced-3d` auto-enable `-bc`.)

> If that is not the case, would sliced 3D also declare the enum values?

This is a good question because it actually impacts how the validation will work exactly. If they are orthogonal features then they need to _both_ enable these formats - i.e. enabling _either_ one will make createTexture() and others pass the `Validate texture format required features` validation step.

I think this is preferable to allowing you to enable `-bc-sliced-3d` but then still having createTexture() throw an exception if you try to use it.

Reply:
@Kangz in case you have opinions

Reply:
@kainino0x Understood, my interpretation was that WebGPU would follow the footsteps of sliced-3d extension for ASTC (I mean keeping it analogous in BC) as found in OpenGL, where, IMO, reasonably, it just enables 3D on pre-definition of base extension: https://registry.khronos.org/OpenGL/extensions/KHR/KHR_texture_compression_astc_sliced_3d.txt

> Requires GL_KHR_texture_compression_astc_ldr

> This extension extends the functionality of GL_KHR_texture_compression_astc_ldr to include slice-based 3D textures

Reply:
@Kangz thank you! I think it would be more consistent to require enabling both separately (and this can relax later in case it is an inconvenience) and just make BC 3D extension a table change just like ASTC sliced 3D as in OpenGL extension:

> Modify the "3D Tex." column to be checked for all ASTC formats

Though I'm also not too strongly opinionated. I think, in any case, formats should be defined in base BC, and then this should just allow the predefined formats to be used as sliced 3D rather than functioning separately. I can update PR depending on @kainino0x suggestion following this.

Reply:
I think I was suggesting a different thing:

1. For this PR, allow enabling `-bc` and `-bc-sliced-3d` independently, each can be used without the other. Later, consider (4).

Other options are:

2. Allow enabling `-bc-sliced-3d` but you can't use the formats without also enabling `-bc`. Later, consider (4).
3. For this PR, error if `-bc-sliced-3d` is enabled without `-bc`. Later, consider (4).
4. Implement auto-enabling now.

1/2 are the most trivial because we don't currently do anything like 3 or 4, but 4 is not that complicated so we _could_ just do it.

Reply:
Mozilla prefers #2 "stacking" -bc-3d on -bc. (and we can totally warn about #3, but I don't think it should error)

If we want #1, we think we should change -bc to -bc-2d. Then it would be clear that -bc-2d enables 2d, and -bc-3d enables 3d.

We don't think we should #4 auto-enable.

Reply:
Meeting: Resolved on option 2 - which means the table as written in this PR is already correct.